### ISSUE TITLE
Two WAI-ARIA attribute issues

### ISSUE THREAD

**Issue by GSSCDylan** [IBM/carbon-components-angular#1879]
## Detailed description
1: The WAI-ARIA attribute 'aria-disabled' is not valid for the element <ibm-combo-box> with WAI-ARIA role 'none'
2: An element with WAI-ARIA role 'combobox' does not have the required WAI-ARIA attribute(s): 'aria-expanded'

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug
> Is this issue related to a specific component?
ibm-combo-box 

> What browser are you working in?
FireFox 90.0.2 (64-bit)
> What version of the Carbon Design System are you using?
    "carbon-components": "10.31.0",
    "carbon-components-angular": "4.46.0",

## Additional information

* Screenshots or code
<img width="1166" alt="combox" src="https://user-images.githubusercontent.com/73825731/128344830-667e0196-6abf-49d4-9a32-73cb2d9bfe04.png">

* Notes

**Comment by Donisius** [comment]
> An element with WAI-ARIA role 'combobox' does not have the required WAI-ARIA attribute(s): 'aria-expanded'

This should already be addressed in #1819 and is included in version `4.52.0`
